core/controllers/blog_dashboard.py

```diff
@@ -121,6 +134,23 @@ def post(self) -> None:
         new_blog_post = blog_services.create_new_blog_post(self.user_id)
         self.render_json({'blog_post_id': new_blog_post.id})
 
+    @acl_decorators.can_access_blog_dashboard
+    def put(self) -> None:
+        """Updates author details of the user."""
+        displayed_author_name = self.normalized_payload.get(
+            'displayed_author_name')
+        author_bio = self.normalized_payload.get('author_bio')
+        blog_services.update_blog_author_details(
+            self.user_id, displayed_author_name, author_bio
+        )
+        author_details = (
+            blog_services.get_blog_author_details(self.user_id).to_dict())
```

Comment:
same comment as above, here also `blog_services.get_blog_author_details(self.user_id)` can return None value ( because in blog_services.py  this method is defined as this method can return none values as well ).

Reply:
please handle the None case of `blog_services.get_blog_author_details(self.user_id)` by raising an exception.

Reply:
Done!